core/templates/dev/head/pages/exploration_player/tutor_card_directive.html

```diff
@@ -37,7 +37,7 @@
     <div ng-show="arePreviousResponsesShown"
          class="conversation-skin-tutor-card-middle-section conversation-skin-responses-animate-slide">
       <div ng-repeat="responsePair in activeCard.inputResponsePairs track by $index">
-        <div ng-if="!$last || !isInteractionInline">
+        <div ng-if="(!$last || !isInteractionInline) && !conceptCardIsBeingShown">
```

Comment:
Just to check -- should the !conceptCardIsBeingShown go one level higher? Not sure you need the "previous responses" card at all in that case.

Reply:
Yeah, done.